src/gossip/gossip-startup.lisp

```diff
@@ -37,7 +37,7 @@
               (make-node :uid pubkey))
             pubkeys)
       ;; clear log and start server
-      (run-gossip-sim :TCP)
+      (run-gossip-sim)
```

Comment:
@svspire The name `run-gossip-sim` is no longer accurate as it no longer refers to a simulation but rather the main loop.

Reply:
Fixed.